doc/source/config.rst

```diff
@@ -258,6 +258,23 @@ as part of the :func:`~satpy.modifiers.angles.get_angles` and
 used by multiple modifiers and composites including the default rayleigh
 correction.
 
+Clipping Negative Infrared Radiances
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Environment variable**: ``SATPY_CLIP_NEGATIVE_RADIANCES``
+* **YAML/Config Key**: ``clip_negative_radiances``
+* **Default**: False
+
+Whether to clip negative infrared radiances to the minimum possible value before
```

Comment:
Can we exchange the word "possible" for "allowable" here please?

Reply:
Done